src/App.js

```diff
@@ -781,54 +771,36 @@ export default class App extends Component {
       voteEndTime,
       voteStartTime
     } = body.contents;
-    const proposals = proposalsList.filter(p => p.proposalId)
-    this.setState(state => ({
-      ...state,
+    const proposals = proposalsList.filter(p => p.proposalId).filter((p, i, list) => list.findIndex(p2 => p2.proposalId === p.proposalId) === i);
```

Comment:
what is the second `filter` call for?

Reply:
It's temporary, but now we have couple of proposal with same id in store. Probably some comment needed here